src/functions-reference/sparse_matrix_operations.Rmd

```diff
@@ -86,6 +86,16 @@ followed by the size of `csr_extract_w(a)` plus one; see section
 [compressed row storage](#CSR).
 `r since("2.8")`
 
+
+<!-- tuple(vector, array[] int, array[] int); csr_extract; (matrix a); -->
+\index{{\tt \bfseries csr\_extract }!{\tt (matrix a): tuple(vector, array[] int, array[] int)}|hyperpage}
+
+`tuple(vector, array[] int, array[] int)` **`csr_extract`**`(matrix a)`<br>\newline
+Return all three components of the CSR representation of the matrix `a`; see
```

Comment:
Same comment---this doesn't really say what the function does directly.

Reply:
These functions all mainly link to the main page on CSR - `extract_w` just says "Return non-zero values in matrix a; see section `[compressed row storage](#CSR)`."

Reply:
If you think that's OK, I'm OK leaving these things indirect.  I'm personally a bit frustrated with doc that sends me reference chasing, especially if they are on different pages of html.